css/properties/background-clip.json

```
@@ -168,10 +168,7 @@
               "chrome": {
                 "version_added": "3",
                 "partial_implementation": true,
-                "notes": [
-                  "This value is supported with the prefixed version of the property only.",
-                  "According to the <a href='https://webkit.org/blog/164/background-clip-text/'>WebKit blog</a>, text decorations or shadows are not included in the clipping."
-                ]
+                "notes": "This value is supported with the prefixed version of the property only."
```

This note is strange, we usually use `prefix` for this, right?

---

It's a bit of a weird situation, because the value itself isn't what gets prefixed.  What this note is trying to describe is that `-webkit-background-clip: text` is supported, but `background-clip: text` is not.

---

Oh yes, sorry I got it now. Slow brain was slow. Rewording this to be extra clear.

```suggestion
                "notes": "The <code>text</code> value is only supported by <code>-webkit-background-clip</code> (and not by <code>background-clip</code>)."
```